internal/provider/provider.go

```diff
@@ -416,6 +416,8 @@ func New(_ context.Context) (*schema.Provider, error) {
 			"aws_acmpca_certificate_authority": acmpca.DataSourceCertificateAuthority(),
 			"aws_acmpca_certificate":           acmpca.DataSourceCertificate(),
 
+			"aws_amp_workspace": amp.DataSourceWorkspace(),
```

Comment:
should we stick to the prometheus nomenclature used in https://github.com/lewinkedrs/terraform-provider-aws/blob/main/internal/provider/provider.go#L932 ?

Reply:
Agreed.

updated to

```
"aws_prometheus_workspace": amp.DataSourceWorkspace(),
```